admin_guide/out_of_resource_handling.adoc

```diff
@@ -47,10 +48,13 @@ description column based on the node summary API.
 |`memory.available` = `node.status.capacity[memory]` - `node.stats.memory.workingSet`
 |===
 
+[NOTE]
+====
 In future releases, the node will support the ability to trigger eviction
 decisions based on disk pressure. Until then, use
 xref:../admin_guide/garbage_collection.adoc#admin-guide-garbage-collection[container
 and image garbage collection].
+====
 
 [[out-of-resource-eviction-thresholds]]
 === Eviction Thresholds
```

Comment:
an example for a soft eviction and hard eviction threshold would be the following:

* operator wants to evict immediately if memory falls below <5%  (hard threshold)
* operator wants to evict if memory falls below <30% for 1 min (soft threshold)

in this scenario, the operator would like their machines to steady at 70% utilization, but are willing to go above that for short periods of time.  that is the general idea.  our ops team will typically use two thresholds for a similar reason.